app/services/user_alerts/alert_user_about_new_device.rb

```diff
@@ -33,7 +33,7 @@ def self.send_alert(user:, disavowal_event:, disavowal_token:)
           'sign_in_unsuccessful_2fa',
           'sign_in_after_2fa',
         ],
-      ).order(:created_at).includes(:device)
+      ).order(:created_at).includes(:device).to_a
```

Comment:
Not worth blocking this fix, but I think we could consider a limit here on the outside chance that there are an incredible number of events.

Reply:
I'll follow-up with you separately on this. I agree with the general premise, but I'd have some concern about limiting what we show here.

Reply:
maybe if there are more we just get the count of the rest and say something like "and _15_ more devices"